src/current/cockroachcloud/plan-your-cluster.md

```diff
@@ -8,163 +8,99 @@ cloud: true
+- **Asymmetric multi-region workloads**: Often, multi-region deployments are asymmetric in terms of CPU utilization across regions. For example, in a three-region cluster, two of the three regions may require more capacity than the third, or vice versa. In CockroachDB {{ site.data.products.standard }}, a cluster's compute capacity is available in all regions.
 
-Given that Standard vCPUs are priced at 60% of Advanced vCPUs, estimating the cost of a workload on Standard as compared to Advanced means estimating the difference in number of vCPUs required between the two plans. For some workloads, Standard requires higher vCPU capacity, for others Advanced, depending on the properties of the workload and how the different underlying CockroachDB deployment architectures provide resources to that workload.
+  For workloads that require symmetric capacity per region or where a given region must maintain a given compute capacity, CockroachDB {{ site.data.products.advanced }} may be a better option.
 
-### vCPU capacity ranges based on Cockroach Labs testing
+- **Workloads that are sensitive to latency**: CockroachDB {{ site.data.products.standard }} clusters use a different networking architecture than CockroachDB {{ site.data.products.advanced }}. This architecture can introduce additional query latency of around 1 microsecond when compared with CockroachDB {{ site.data.products.advanced }}.
 
-In our internal testing, running a wide range of workloads on both Standard and Advanced, we’ve seen the following results:
+- **Workloads with advanced security or compliance requirements**: For workloads subject to advanced security or compliance requirements, consider CockroachDB {{ site.data.products.advanced }}, which complies with [PCI DSS and HIPAA]({% link cockroachcloud/pci-dss.md %}). To learn more refer to [Compliance in CockroachDB {{ site.data.products.cloud }}]({% link cockroachcloud/compliance.md %}).
 
-Outcome: For 75% of all all tested workloads, the required Standard total cluster cost (in $ per month) is within 40% \- 70% of the total Advanced cluster cost (in $ per month).
+For more details to help you compare CockroachDB {{ site.data.products.cloud }} cluster plans, refer to [CockroachDB {{ site.data.products.cloud }} Overview]({% link cockroachcloud/index.md %}).
 
-The following histogram provides more detail on the test outcomes.  
-![][image3]
+## Storage pricing
 
-## Examples and Benchmarks for vCPU Capacity
+Storage capacity for a CockroachDB {{ site.data.products.standard }} cluster is [priced]({% link cockroachcloud/billing-management.md %}) on demand in units of GiB-month. Storage prices vary across providers and [regions]({% link cockroachcloud/regions.md %}). Storage is charged based on the logical amount of data in the database.
+- **3 replicas included**: - CockroachDB {{ site.data.products.standard }} maintains at minimum three replicas of your data at no additional cost.
```

Comment:
Similar bullet fixes needed in this section
<img width="864" alt="Screenshot 2024-09-25 at 1 10 59 AM" src="https://github.com/user-attachments/assets/74ff4ec1-dcea-432e-8b8d-a7ee4e90ea3f">